test/test.config.js

```diff
@@ -6,8 +6,6 @@ const path = require('path');
 
 config.mocha.tests.push(path.join(__dirname, 'mocha'));
 
-config.jsonld.strictSSL = false;
```

Comment:
In connection with this, when we add `bedrock-https-agent` to a lib, we need to set the `rejectUnauthorized` flag to false.  This would become evident later if any HTTP requests were actually made in the test suite.  Apparently that is not happening now.

Reply:
https-agent is in the package.json so I added the settings for it: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity-storage/pull/6/commits/ed8af375d7ac6191ab2511a8fd11a572aa3d4d5b